controller/src/lib.rs

```diff
@@ -297,7 +297,7 @@ impl Controller {
     // Return a header using the next available message ID.
     fn next_header(&self) -> Header {
         Header {
-            version: message::version::V1,
+            version: message::version::V2,
```

Comment:
On thing that occurs to me is we might want to add a `message::version::CURRENT` const "alias", so that this library is always compiled against the latest version in the source tree. Thoughts?

Reply:
Yeah, that makes sense to me – done.